src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,73 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String RESULT = "result";
+    private static final String COMMA = ",";
+    private static final int TWO = 2;
+    private static final int ZERO = 0;
+    private static final int ONE = 1;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        calculateReport(readFromFile(fromFileName), toFileName);
```

Comment:
Don't complicate your code
```suggestion
        int[] output = readFile(fromFileName);
        String result = calculateReport(output);
        writeToFile(result, toFileName);
```